tasks/src/image-segmentation/data.ts

```diff
@@ -59,6 +59,10 @@ const taskData: TaskDataCustom = {
 			description: "A semantic segmentation application that predicts human silhouettes.",
 			id:          "keras-io/Human-Part-Segmentation",
 		},
+		{
+			description: "Cell Instance Segmentation - Sartorius Kaggle Competition",
```

Comment:
```suggestion
			description: "An instance segmentation application to predict neuronal cell types from microscopy images.",
```
Does this look good?

Reply:
Yes perfect, Thank you very much @merveenoyan.